editor/src/clj/editor/protobuf.clj

```diff
@@ -394,7 +398,8 @@ Macros currently mean no foreseeable performance gain however."
                   (keep (fn [^Method m]
                           (let [m-name (.getName m)
                                 set-via-builder? (and (.startsWith m-name "set")
-                                                   (some-> m (.getParameterTypes) ^Class first (.getName) (.endsWith "$Builder")))]
+                                                      (when-some [^Class first-arg-class (first (.getParameterTypes m))]
+                                                        (.endsWith (.getName first-arg-class) "$Builder")))]
```

Comment:
Fixed release-only reflection warning. Exotic! 😄

Reply:
Hmm, how is it release-only?

Reply:
It is really strange, but it appears the `:aot` compiler works slightly differently. It was getting confused by the `^Class` tag in the `some->` macro - the type hint was not being applied to the result of the `first` call.